topics/metabolomics/tutorials/msi-analyte-distribution/workflows/main_workflow.ga

```diff
@@ -1,479 +1,407 @@
 {
-  "uuid": "f6afebc9-9c98-45e2-b79b-da2effab6f0d",
-  "tags": [
```

Comment:
if you restore this `tags` attribute the tests will pass again :+1:

Also, thanks for your contributions! Please add yourself to the `CONTRIBUTORS.yaml` file so that you will be included in our [contributor hall of fame](https://training.galaxyproject.org/training-material/hall-of-fame) :)